src/Infrastructure/BotSharp.OpenAPI/Controllers/KnowledgeController.cs

```diff
@@ -11,11 +11,12 @@ namespace BotSharp.OpenAPI.Controllers;
 public class KnowledgeController : ControllerBase, IApiAdapter
 {
     private readonly IKnowledgeService _knowledgeService;
-    public KnowledgeController(IKnowledgeService knowledgeService)
+    private readonly IPdf2TextConverter _pdf2TextConverter;
```

Comment:
The converter only be used in `FeedKnowledge`, so we can get this instance in `FeedKnowledge` function internally. Don't need to declare it this class level.